### PR TITLE
Code review best practices

### PR DESCRIPTION
**Pull request by rmacken98**
Added Code Review best practices 
Added quick links + references
Added Images to best practices
Fixed spelling mistake in code review bad practices.

### CODE REVIEW

**Comment by Lordjiggyx** [comment]
Hey ronan just a few things i spotted
1. The links in thr code review Bad practices file don't work
2. The code review Home page should have links to best practices and bad practices
3. The images in best Practices arent working 

Try get these fixed and when done im happt for it to merge

**Review by Lordjiggyx — CHANGES_REQUESTED** [review]
Hey ronan just a few things i spotted

The links in thr code review Bad practices file don't work
The code review Home page should have links to best practices and bad practices
The images in best Practices arent working
Try get these fixed and when done im happt for it to merge

File: CodeReviews/CodeReviewBestPractices.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+* Do unit or end-to-end tests need to be updated/created?
+
+
+### Speed of a code review and how long should be spent on a given review
+ 
+ #### 60-90 minutes max per review
+ A code reviewer should take the time to carry out a slow and thorough review. However this should take no more than 60-90 minutes according to study's carried out by IBM and others it has been shown that after 60 minutes reviewers tend to get tired and stop finding additional defects/issues with the code.
+
+ <p align="center">
+<img src="images/image002.png" alt="Defects Density vs Lines of code per hour" width="400">
```
Comment (NicBoi) [inline CodeReviews/CodeReviewBestPractices.md:44]:
```suggestion
<img src="Images/image002.png" alt="Defects Density vs Lines of code per hour" width="400">
```

File: CodeReviews/CodeReviewBestPractices.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+# Best Practices 
+
+ <p align="center">
+<img src="images/cleancodereview.png" alt="Defects Density vs Lines of code per hour" width="400">
```
Comment (NicBoi) [inline CodeReviews/CodeReviewBestPractices.md:4]:
```suggestion
<img src="Images/cleancodereview.png" alt="Defects Density vs Lines of code per hour" width="400">
```

**Review by NicBoi — CHANGES_REQUESTED** [review]

**Comment by NicBoi** [comment]
Just noticed now Ronan, you're merging from your forked repository, instead you should push the branch to this repository and then create a PR

**Comment by NicBoi** [comment]
> Hey ronan just a few things i spotted
> 
> The links in thr code review Bad practices file don't work
> The code review Home page should have links to best practices and bad practices
> The images in best Practices arent working
> Try get these fixed and when done im happt for it to merge

This is probably due to the fact that Ronan is merging from his forked repository and for some reason even though the paths are relative they point to the master branch.

**Comment by rmacken98** [comment]
> > Hey ronan just a few things i spotted
> > The links in thr code review Bad practices file don't work
> > The code review Home page should have links to best practices and bad practices
> > The images in best Practices arent working
> > Try get these fixed and when done im happt for it to merge
> 
> This is probably due to the fact that Ronan is merging from his forked repository and for some reason even though the paths are relative they point to the master branch.

Thanks Nic, I'll close the PR, make the changes and push to this repository instead

**Comment by rmacken98** [comment]
> Hey ronan just a few things i spotted
> 
> The links in thr code review Bad practices file don't work
> The code review Home page should have links to best practices and bad practices
> The images in best Practices arent working
> Try get these fixed and when done im happt for it to merge

Making the changes now and making a new pr pushed directly to branch not from my forked repo. Thanks Tomi.

**Pull request closed without merging**